### PR TITLE
Update UI to use BorderLayout, fixes resizeable issues

### PR DESCRIPTION
**Pull request by KronosDesign**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java [hunk 1]
```diff
@@ -37,10 +38,13 @@
 public final class ClientUI extends JFrame
 {
 	private JPanel container;
+	private JPanel navContainer;
 	private ClientPanel panel;
 	private NavigationPanel navigationPanel;
 	private PluginPanel pluginPanel;
 	private boolean expanded;
+	private int collapsedWidth = ClientPanel.PANEL_WIDTH + NavigationPanel.PANEL_WIDTH;
+	private int expandedWidth = collapsedWidth + PluginPanel.PANEL_WIDTH;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java]: These two fields seem constant, should be static final

**Comment by Adam-** [comment]
This seems to move the nav bar to be on the left side of the plugin area instead of the right side, which I don't want.

**Comment by RobinWeymans** [comment]
And do you want the ClientPanel to change size when opening and closing the PluginPanel?

**Comment by KronosDesign** [comment]
@RobinWeymans I will eventually revisit the entire UI but for now it works well enough.
Ideally, i'd like to create a slide-out style menu that would sit on top of the ClientPanel.

**Comment by Adam-** [comment]
Had to modify this to slightly to make it not resize the height down smaller than the panel height (I don't know why) when closing the side panel... Otherwise seems to work, merged.

**Pull request closed without merging**